File: LinkedList.py
class ListNode:
	def __init__(self, val = None):
		self.val = val
		self.next = None

class LinkedList:
	def __init__(self):
		self.head = ListNode()

	def add(self, value):
		node = ListNode(value)
		node.next = self.head.next
		self.head.next = node

	def remove(self):
		self.head.next = self.head.next.next

	def print(self):
		cur = self.head.next
		while(cur != None):
			print(cur.val)
			cur = cur.next

linked_list = LinkedList()
linked_list.add(1)
linked_list.add(3)
linked_list.add(5)
linked_list.add(7)
linked_list.remove()
linked_list.print() # 5,3,1

File: linked_list.js
// Node
class Node {
    constructor(data) {
		this.data = data;
		this.next = null;
    }
}

// What you need to do
class LinkedList {
	constructor() {
		this.head = null;
	}

	append(data) {
		if (this.head == null) {
			this.head = new Node(data);		
		} else {
			let cur = this.head;
			while(cur.next != null) {
				cur = cur.next;
			}
			cur.next = new Node(data);	
		}
	}

	print() {
		let cur = this.head;
		while(cur) {
			console.log(cur.data);
			cur = cur.next;
		}	
	}

}

let linked_list = new LinkedList()
linked_list.append(1)
linked_list.append(2)
linked_list.append(3)
linked_list.print()

